src/app/public/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -191,7 +207,10 @@ export class SkyColorpickerInputDirective
       });
 
     this.skyColorpickerInput.updatePickerValues(this.initialColor);
-    this.skyColorpickerInput.backgroundColorForDisplay = this.initialColor;
+
+    if (!this.disabled) {
+      this.skyColorpickerInput.backgroundColorForDisplay = this.initialColor;
+    }
```

Comment:
if the colorpicker component's input is disabled on initialization, do no set the initial value